### ISSUE TITLE
Renaming local files with spaces (vd file browser) throws UnboundLocalError

### ISSUE THREAD

**Issue by layertwo** [saulpw/visidata#242]
I'm following the Youtube video for manipulating files in a local directory, and when running it against 1.5.1, I'm getting console errors.

I'm intending to replace all the spaces in the filenames, but am unable to perform that action.

Steps:

1. Open directory with files with spaces (note here: I'm doing that via `vd .` since I'm already in the directory.
2. g*
3.  \ (that's space and \)
4. Error

Here's the error message I'm receiving:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/visidata/vdtui.py", line 650, in toplevelTryFunc
    ret = func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/visidata/regex.py", line 53, in setValuesFromRegex
    transform = regexTransform(col, rex)
UnboundLocalError: local variable 'col' referenced before assignment
```

**Comment by saulpw** [comment]
Yes, this is a known bug in 1.5.1 (has to do with `g*`, not the spaces).  We'll be releasing 1.5.2 soon to correct this issue.

**Comment by anjakefala** [comment]
Confirmed that bug was fixed in https://github.com/saulpw/visidata/commit/8a72d576ff9d7e99f477c7df062a0f27c107a368 and the fix will be v1.5.2.
